pkgs/applications/video/jellyfin-media-player/default.nix

```
@@ -74,6 +75,8 @@ mkDerivation rec {
     "-DCMAKE_BUILD_TYPE=Release"
     "-DQTROOT=${qtbase}"
     "-GNinja"
+  ] ++ lib.optional (!withDbus) [
```

```suggestion
  ] ++ lib.optionals (!withDbus) [
```

This creates a nested list.

```console
nix-repl> [ "moo" ] ++ lib.optional true [ "oink" ]
[ "moo" [ ... ] ]
```

I think you want `lib.optionals` instead.

```console
nix-repl> [ "moo" ] ++ lib.optionals true [ "oink" ]
[ "moo" "oink" ]
```

---

Understood, thanks!